pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "evergreen.py"
-version = "3.6.5"
+version = "3.6.7"
```

Comment:
The version is `3.6.7` because I want to merge it after https://github.com/evergreen-ci/evergreen.py/pull/143